### PR TITLE
feat: add consume ActorNameChanged event

### PR DESCRIPTION
**Pull request by Mech0z**
## Description

This changes the name of the Market Participant based on the received event

## References

* #1732

### CODE REVIEW

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/MarketParticipants/Handlers/Mappers/MarketParticipantIntegrationEventMapper.cs [hunk 1]
```diff
@@ -77,5 +77,13 @@ public static MarketParticipantB2CActorIdChangedCommand Map(ActorExternalIdChang
                 gridUpdatedIntegrationEvent.GridAreaId,
                 gridUpdatedIntegrationEvent.GridAreaLinkId);
         }
+
+        public static MarketParticipantNameChangedCommand Map(
```
Comment (prtandrup) [inline source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/MarketParticipants/Handlers/Mappers/MarketParticipantIntegrationEventMapper.cs:81]: Think we are missing this mapping in `MarketParticipantIntegrationEventMapperTests`

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Domain/MarketParticipants/MarketParticipant.cs [hunk 2]
```diff
@@ -137,5 +137,10 @@ public void UpdateB2CActorId(Guid? b2CActorId)
         {
             B2CActorId = b2CActorId;
         }
+
+        public void UpdateName(string name)
```
Comment (prtandrup) [inline source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Domain/MarketParticipants/MarketParticipant.cs:141]: Please add test in `MarketParticipantTests`

**Review by prtandrup — APPROVED** [review]
LGTM with a couple of test comments

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/MarketParticipants/Handlers/Mappers/MarketParticipantIntegrationEventMapper.cs (hunk 1 above)
Comment (Mech0z) [reply source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/MarketParticipants/Handlers/Mappers/MarketParticipantIntegrationEventMapper.cs:81]: added https://github.com/Energinet-DataHub/geh-charges/pull/1798/commits/7ef420cc4031a5fd562db22a12ee9682f4feb0cf

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Domain/MarketParticipants/MarketParticipant.cs (hunk 2 above)
Comment (Mech0z) [reply source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Domain/MarketParticipants/MarketParticipant.cs:141]: added https://github.com/Energinet-DataHub/geh-charges/pull/1798/commits/7ef420cc4031a5fd562db22a12ee9682f4feb0cf

**Pull request merged**